Demo/UITests/Helpers.swift

```diff
@@ -1,57 +1,38 @@
 import XCTest
 
 extension XCTestCase {
-    func waitForElementToAppear(_ element: XCUIElement, timeout: TimeInterval = 10,  file: String = #file, line: Int = #line) {
+
+    func waitForElementToAppear(_ element: XCUIElement, timeout: TimeInterval = 20) {
```

Comment:
I hesitate slightly about the bump to 20s. It feels like if it takes over 20s to load, then our test are actually not good

Reply:
I would still love to move to snapshot tests, they're quick and reliable. But it seems pretty far off on our priority list right now.

Reply:
We default to `15` on `braintree_ios` but I know even there sometimes our UITests time out. I think it has more to do with the runners than anything we do. If they take less than 20 seconds it won't wait around though so that's nice. More just giving the runners extra time if they needed it. I'm fine matching BT core if we want but again, we may have to kick them often like we do on BT core.

Reply:
Huge ➕ on moving to snapshot testing. I'm down to mess around with it one open dev if you're down to explore it more. I've not dealt with it much but down to explore!

Reply:
True good call - a lot of it must boil down to GH Actions resources. Down to mess with Snapshot testing sometime! I've worked with a bit prior. Another cool thing you can do is UI test specific components (such as individual buttons and subviews).